cluster-autoscaler/cloudprovider/clusterapi/deployment.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-autoscaler
+  namespace: kube-system
+  labels:
+    app: cluster-autoscaler
+spec:
+  selector:
+    matchLabels:
+      app: cluster-autoscaler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cluster-autoscaler
+    spec:
+      containers:
+      - image: us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.18.1
+        name: cluster-autoscaler
+        command:
+        - /cluster-autoscaler
+        args:
+        - --cloud-provider=clusterapi
+        - --namespace=default
+        - --scale-down-enabled=true
+        - --scale-down-delay-after-add=10s
+        - --scale-down-delay-after-delete=10s
+        - --scale-down-delay-after-failure=10s
+        - --scale-down-unneeded-time=23s
+        - --balance-similar-node-groups
+        - --max-node-provision-time=10m
+        - --expendable-pods-priority-cutoff=-10
+        - --max-nodes-total=24
```

Comment:
I believe we should probably use a variable for this as well, since this will likely need to be tweaked depending on the scale of workload clusters a user might be deploying.